lua/gitlinker/actions.lua

```diff
@@ -26,4 +26,9 @@ local M = {
   system = system,
 }
 
+local clipboard_override = require('gitlinker.configs').get()['clipboard_override']
+if clipboard_override then
+  M.clipboard = clipboard_override
```

Comment:
hi @kraftnix ,

This part is like a cycle-dependency between "gitlinker.actions" and "gitlinker.configs".

I moved this part to "setup" here: https://github.com/linrongbin16/gitlinker.nvim/pull/280/files#diff-e85b0a73f385d3b9be62752dbc0afd2ca8eba8b75c12d7667a5514b8b35755ee